experimental/GModule/GModule.jl

```diff
@@ -20,7 +21,18 @@ function GAP.gap_to_julia(::Type{QabElem}, a::GAP.GapObj) #which should be a Cyc
   return z
 end
 
+function GAP.gap_to_julia(::Type{QabElem}, a::Integer)
```

Comment:
```suggestion
function GAP.gap_to_julia(::Type{QabElem}, a::GAPInt)
```
Otherwise you won't capture big GAP integers.